xmake/modules/private/xrepo/action/install.lua

```diff
@@ -79,7 +79,8 @@ function menu_options()
                                        "    - xrepo install -p iphoneos -a arm64 \"zlib >=1.2.0\"",
                                        "    - xrepo install -p android [--ndk=/xxx] -m debug \"pcre2 10.x\"",
                                        "    - xrepo install -p mingw [--mingw=/xxx] -k shared zlib",
-                                       "    - xrepo install conan::zlib/1.2.11 vcpkg::zlib"}
+                                       "    - xrepo install conan::zlib/1.2.11 vcpkg::zlib",
+                                        values = function (complete, opt) return import("private.xrepo.quick_search.completion")(complete, opt) end}
```

Comment:
it will break `xrepo install -h`

> error: attempt to index a boolean value (local 'complete')

you need handle help menu and disable complete.

```lua
if complete then
    -- do complete
end
```